Scripts/auth.py

```diff
@@ -44,16 +48,34 @@ def get_authenticated_service():
   API_VERSION = 'v3'
   DISCOVERY_SERVICE_URL = "https://youtube.googleapis.com/$discovery/rest?version=v3" # If don't specify discovery URL for build, works in python but fails when running as EXE
 
-  # Check if client_secrets.json file exists, if not give error
+  # Check if client_secrets.json file exists, if not give error and the option to add the file via a filepicker
   if not os.path.exists(CLIENT_SECRETS_FILE):
     print(f"\n         ----- {F.WHITE}{B.RED}[!] Error:{S.R} client_secrets.json file not found -----")
     print(f" ----- Did you create a {F.YELLOW}Google Cloud Platform Project{S.R} to access the API? ----- ")
     print(f"  > For instructions on how to get an API key, visit: {F.YELLOW}TJoe.io/api-setup{S.R}")
     print(f"\n  > (Non-shortened Link: https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Instructions:-Obtaining-an-API-Key)")
-    input("\nPress Enter to Exit...")
-    sys.exit()
+    print(f"\n")
+    print(f"\n > If a file already exists, please input 'Y' or 'Yes'")
+    print(f"\n > Otherwise press Enter to exit")
+    
+    inp = input("\nInput: ")
+    if inp.upper() in ("Y","YES"):
+      root = tk.Tk()
+      root.withdraw()
+      #try and except is used because a lot of errors occur if the file picker is closed without picking a file.
+      try:
+        file_path = filedialog.askopenfilename()
+        #replaces the path for the file to wherever the currrent path is, also renames the file
+        os.replace(file_path, f"{os.getcwd()}/client_secrets.json")
```

Comment:
I'm not sure how I feel about moving the file to the current directory -- if this is made into a package, this would definitely need to change
Not an issue related to this PR though, better to make a standalone issue for this

Reply:
Would likely require a rewrite of the file -- if a file picker (or config) is used, just leave the file where it is 😁
Automatically generating a secrets file in the current dir is more expected functionality, in my opinion